cli/src/main/java/org/owasp/dependencycheck/App.java

```diff
@@ -66,6 +66,10 @@ public class App {
      * Properties file error message.
      */
     private static final String ERROR_LOADING_PROPERTIES_FILE = "Error loading properties file";
+    /**
+     * System specific new line character.
+     */
+    private static final String NEW_LINE = System.getProperty("line.separator", "\n").intern();
```

Comment:
I see no need for interning the String.

```suggestion
    private static final String NEW_LINE = System.getProperty("line.separator", "\n");
```

Reply:
no clue why this appeared twice